bundles/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/handlers/ParadoxIP150BridgeHandler.java

```diff
@@ -144,23 +145,20 @@ private synchronized void doPostOnlineTask(IParadoxInitialLoginCommunicator init
         scheduler.schedule(() -> createDiscoveredCommunicatorJob(panelType), 3, TimeUnit.SECONDS);
     }
 
+    // test line endings #3
```

Comment:
What is this comment about?

Reply:
Good that you noticed it. I will remove it. Thanks !